consumer/pdata/common.go

```diff
@@ -354,6 +354,10 @@ func NewAttributeMap() AttributeMap {
 	return AttributeMap{&orig}
 }
 
+func NewAttributeMapFromKeyValues(orig *[]otlpcommon.KeyValue) AttributeMap {
+	return newAttributeMap(orig)
+}
```

Comment:
We should not expose any public API that depends on the proto definition.

Reply:
Ok, I added some new functions in `goldendataset` to build the `AttributeMap`.